GliaWidgets/Theme/Survey/Theme+Survey.swift

```diff
@@ -215,4 +104,21 @@ extension UIFont {
         default:        return 0.2
         }
     }
+
+    static func convertToFont(font: RemoteConfiguration.Font?) -> UIFont? {
```

Comment:
Will be better to move it to separate file. For example to `GliaWidgets/Lib/Extensions`

Reply:
I moved the static function to a separate file.